### DISCUSSION TITLE
Local server issues/drops

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1081] [Help]
For the first time as I was travelling and I needed to user AG locally on my laptop.

I noticed several issues that doesnt exists when working with remote server :

- when plugin GUI are open sometimes I get cpu spikes and drops, making it unusable. Closing AG insert solve the issue and cpu usage is normal. Never got that with remote server.

- when I use a plugin GUI, the AG gui disappear, which is really painfull.  Only the used plugin is displayed. I need to click again on the AG insert to get it back.

- I noticed a lot of server crashes, especially when multiple plugins are opened. Again using the exact same session but with a remote server everything is stable.

Config : Studio One 6 M1 native / mac

Any idea ?

**Comment by apohl79** [comment]
> - when plugin GUI are open sometimes I get cpu spikes and drops, making it unusable. Closing AG insert solve the issue and cpu usage is normal. Never got that with remote server.

Are any specific plugin(s) causing this?

> - when I use a plugin GUI, the AG gui disappear, which is really painfull.  Only the used plugin is displayed. I need to click again on the AG insert to get it back.

Sounds like some DAW behavior? Maybe there is an 'always on top' option in S1? 

> - I noticed a lot of server crashes, especially when multiple plugins are opened. Again using the exact same session but with a remote server everything is stable.

I assume that some plugins are causing those crashes. Maybe you can isolate those issues to failing plugins?
